webpack.config.js

```diff
@@ -49,45 +49,46 @@ const RULES = {
       {
         loader: 'awesome-typescript-loader',
         options: {
-          target: 'es2017',
-        },
-      },
-    ],
-  },
-}
+          target: 'es2017'
+        }
+      }
+    ]
+  }
+};
 
 const config = (env = DEFAULT_ENV) => {
-  const { ifProd, ifNotProd } = getIfUtils(env)
+  const { ifProd, ifNotProd } = getIfUtils(env);
   const PLUGINS = removeEmpty([
     // enable scope hoisting
     new webpack.optimize.ModuleConcatenationPlugin(),
     // Apply minification only on the second bundle by using a RegEx on the name, which must end with `.min.js`
-    ifProd(
-      new UglifyJsPlugin({
-        sourceMap: true,
-        compress: {
-          screw_ie8: true,
-          warnings: false,
-        },
-        output: { comments: false },
-      })
-    ),
+    //   ifProd(
+    //     new UglifyJsPlugin({
+    //       sourceMap: true,
+    //       uglifyOptions: {
+    //         compress: {},
```

Comment:
Updated these to match the new uglify options with newest webpack, but it didn't quite work.